packages/gatsby-theme-code-recipes/gatsby-config.js

```diff
@@ -7,5 +7,12 @@ module.exports = ({ path = 'src/recipes' } = {}) => ({
         path,
       },
     },
+    {
+      resolve: `gatsby-plugin-typescript`,
+      options: {
+        isTSX: true,
```

Comment:
Hey @velein, why do we need `isTSX` and `allExtensions` here?

Reply:
The more I read about these two options, the more I'm sure I'll actually turn them on in my personal projects, but I'm just not sure if we want to force them for `gatsby-theme-code-recipes` users.

Reply:
Fair point, just removed those